web/modules/custom/solution/solution.behat.inc

```diff
@@ -96,6 +96,15 @@ class SolutionSubContext extends DrupalSubContextBase implements DrupalSubContex
     $this->visitPath($path);
   }
 
+  /**
+   * Navigates to the solution overview page.
+   *
+   * @When (I )visit the solution overview( page)
```

Comment:
The `I` should be outside the parenthesis. The `When/Then/But/Given/And visit the solution overview` is wrong.

Reply:
OK I was following the same practice as we used elsewhere but this has also been rubbing me the wrong way for a long time. I will update it everywhere.